crates/bevy_gltf/src/lib.rs

```diff
@@ -63,3 +69,52 @@ pub struct GltfPrimitive {
     pub mesh: Handle<Mesh>,
     pub material: Option<Handle<StandardMaterial>>,
 }
+
+/// Part of a [`GltfNodeAnimation`], interpolation method for an animation.
```

Comment:
First what it is, then context.
```suggestion
/// Interpolation method for an animation. Part of a [`GltfNodeAnimation`].
```